agent/Makefile

```
@@ -1,6 +1,6 @@
 # Makefile for iconfucius Python package
 
-.PHONY: help install install-dev install-blst build clean publish-test publish test lint \
+.PHONY: help install install-dev install-blst bundle-ui build clean publish-test publish test lint \
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)